crates/git/src/repository.rs

```diff
@@ -891,20 +892,21 @@ impl GitRepository for RealGitRepository {
 
     fn fetch(
         &self,
+        remote_name: String,
```

Comment:
We should switch this to a `FetchOptions` enum, which takes a specific remote or `All`